ui/src/identity.ts

```diff
@@ -29,13 +31,8 @@ export interface Identity {
 const creationStore = remote.createStore<Identity>();
 export const store = creationStore.readable;
 
-interface CreateInput {
-  handle: string;
-  passphrase: string;
```

Comment:
I'm presuming the `passphrase` was un-needed? What the hell is it doing there in the first place?! :joy:

Reply:
that's right. I don't know :shrug: I tested the whole onboarding flow to be super sure. but if you follow the code you see that we first call the endpoint that creates a keystore and then we call the endpoint that creates the identity.

Reply:
It used to be a single call, was changed in multiple iterations of the session feature.